frontend/package.json

```diff
@@ -24,7 +24,8 @@
     "check": "npm run lint && npm run build && npm run test:unit --workspace=rotki",
     "check:all": "npm run lint && npm run build && npm run test:unit --workspace=rotki && npm run test:integration-ci --workspace=rotki",
     "clean:modules": "rimraf node_modules app/node_modules common/node_modules",
-    "dev": "node start-dev.js"
+    "dev": "node start-dev.js",
+    "start": "npm run electron:serve -w rotki"
```

Comment:
`npm run dev` is supposed to be the simpler way to start everything you need to develop the application.

It will also start the Typescript compiler in watch mode for the common library, and the dev proxy that you can use optionally, but you don't have to.

Is there something that doesn't work with `npm run dev` that requires the introduction of a new script?

Reply:
If it is about the ease of calling `npm start` instead of `npm run dev` maybe it would make sense to make `npm start` a shortcut for `npm run dev`.

Reply:
This is not really introducing anything new; it's just simplifying what's already there.  The point is that the existing documentation says to run `npm run electron:serve -w rotki`, and this is not easy to remember or friendly to newcomers.  If `npm run dev` is a direct replacement for `npm run electron:serve -w rotki`, then the docs should be changed to reflect that, since currently `npm run dev` is not documented anywhere AFAICS.

Or if it is not a direct replacement, then this PR is a way to make it easier to remember how to achieve the same effect.

Reply:
Yes, the documentation is indeed outdated. 

`npm run dev` is what a new developer should run since its purpose is to bring up everything needed to develop the application. It is the direct replacement for `npm run electron:serve -w rotki`. 

If you think aliasing `npm run dev` under `npm start` makes things easier for people we can also do that.

Reply:
`npm run dev` is fine by itself.  The most important thing is that the docs are up to date, and easy to follow and remember :-)